layouts/index.html

```diff
@@ -213,14 +213,10 @@ <h3 class="mt-6 text-white">Install on Windows</h3>
                     <p>Use the Windows <a href="https://docs.rapids.ai/install#wsl2" target="_blank">WSL2 installation
                             instructions <i class="fa-solid fa-arrow-up-right"></i> </a></p>
 
-                    <h2 class="mt-9 text-white"><i class="fa-light fa-gears"></i> Do I need an Advanced Install?</h2>
-                    <p> If you want to use Docker, WSL2, have an uncommon OS, hardware configuration, environment, or
-                        need specific versions such as <code> CUDA 11.8 </code> or <code> Python 3.8 </code> then you
-                        should use the
-                        installation selector in our
-                        <a href="https://docs.rapids.ai/install" target="_blank">Advanced Installations Page <i
-                                class="fa-solid fa-arrow-up-right"></i> </a>
-                    </p>
+                    <h2 class="mt-9 text-white"><i class="fa-light fa-gears"></i>RAPIDS Release Selector</h2>
```

Comment:
For visual consistency with the other section headings, we need a space here.
```suggestion
                    <h2 class="mt-9 text-white"><i class="fa-light fa-gears"></i> RAPIDS Release Selector</h2>
```